views/default/elements/forms.css.php

```diff
@@ -115,6 +127,21 @@
 	padding-right: 1rem;
 }
 
+.elgg-color-box {
```

Comment:
why was this added?

Reply:
This was added to show the color box for color.php input

Reply:
![image](https://user-images.githubusercontent.com/2772844/61233248-c58a3400-a74d-11e9-8ed3-15b48a5339b7.png)